pkg/server/server.go

```diff
@@ -335,6 +343,24 @@ func (s *Server) Run(ctx context.Context) error {
 		return nil
 	})
 
+	// TODO(ncdc): eventually remove/replace this. It's needed for the moment because anything/everything that works
+	// with CRDs and CRs now needs to have the APIBindings CRD installed (our custom CRD lister uses the APIBindings
+	// lister).
+	s.AddPostStartHook("kcp-hack-install-apibindings-into-local-admin", func(ctx genericapiserver.PostStartHookContext) error {
+		if err = configcrds.Create(
+			goContext(ctx),
+			apiextensionsClusterClient.Cluster(genericcontrolplane.LocalAdminCluster).ApiextensionsV1().CustomResourceDefinitions(),
+			metav1.GroupResource{Group: apis.GroupName, Resource: "apibindings"},
```

Comment:
isn't this the same as `kcp-install-api-resource-controller`?

Reply:
Yes, but some e2es don't run will that controller enabled, and we now need apibindings available 100% of the time for CRD list/get (without this, informers don't sync in all cases). I did this before consolidating system CRD bootstrapping. Could do a follow-up to consolidate even more.

Reply:
done